app/src/main/java/co/condorlabs/customcomponents/customedittext/EditTextPhoneField.kt

```diff
@@ -50,17 +78,24 @@ class EditTextPhoneField(context: Context, attrs: AttributeSet) :
 
     private fun setPhoneMask() {
         this.editText?.apply {
-            addTextChangedListener(PhoneNumberTextWatcherMask(this))
+            addTextChangedListener(PhoneNumberTextWatcherMask(mask) {
+                try {
+                    setSelection(it)
+                } catch (t: Exception) {
```

Comment:
don't catch all the exception only to throw one especificó

Reply:
Changed